File: apps/ios/Pebbles/Features/Path/Render/Wobble/WobbleFlags.swift
/// Feature gate for the petroglyph wobble experiment (issue #555).
///
/// On in every configuration since #727: the valence fan picker is built out
/// of the wobbled silhouettes, so a Release build that fell back to the smooth
/// `SVGView` path would ship a different picker than the one that was
/// designed. Android already bakes the wobble into its internal-testing
/// releases (decision log, 2026-07-14); this is iOS catching up.
///
/// Kept as a flag rather than deleted so the experiment can still be switched
/// off in one place. Deleting it means removing the Wobble folder and
/// reverting the flag-gated call sites.
enum WobbleFlags {
    static let isEnabled = true
}

File: apps/android/app/src/main/kotlin/app/pbbls/android/features/path/render/wobble/WobbleFlags.kt
package app.pbbls.android.features.path.render.wobble

import app.pbbls.android.BuildConfig

/**
 * Feature gate for the petroglyph wobble experiment (issue #555) — mirrors
 * iOS `WobbleFlags.swift`.
 *
 * Debug-only by construction: the spike evaluates the hand-carved look across
 * every pebble surface in dev builds, and `BuildConfig.DEBUG` is a constant
 * `false` in Release (R8 strips the gated branches) so it can never ship by
 * accident. Deleting the experiment means removing this folder and reverting
 * the flag-gated call sites ([app.pbbls.android.features.path.render
 * .PebbleStaticRender], [app.pbbls.android.features.path.render
 * .PebbleOutlineBackdrop], [app.pbbls.android.features.path.render
 * .GlyphImage]).
 */
internal object WobbleFlags {
    val isEnabled: Boolean = BuildConfig.DEBUG
}
